js/docgist.js

```diff
@@ -20,18 +16,39 @@ function DocGist($) {
         $gistId.keydown(gist.readSourceId);
     });
 
-    function renderContent(content, link) {
+    function getAsciidoctorOptions() {
+        return Opal.hash2(['to_file', 'safe', 'attributes'], {
+            'to_file': false,
+            'safe': 'secure',
+            'attributes': ASCIIDOCTOR_DEFAULT_ATTRIBUTES.concat([].slice.apply(arguments))
+        });
+    }
+
+    function renderContent(content, link, imageBasePath) {
         $('#gist-link').attr('href', link);
         $content.empty();
         var doc, html = undefined;
-        var highlightSource = false;
+        var highlighter = undefined;
         try {
-            doc = Opal.Asciidoctor.$load(content, ASCIIDOCTOR_OPTIONS);
-            var attributeEq = doc.$method('attr?');
-            if (attributeEq.$call('source-highlighter', 'highlightjs') || attributeEq.$call('source-highlighter', 'highlight.js')) {
-              highlightSource = true;
+            doc = Opal.Asciidoctor.$load(content, getAsciidoctorOptions('parse_header_only=true'));
+            console.log(doc);
```

Comment:
Perhaps only log the document when a special debug mode is set, perhaps via the URL?